code/obj/item/misc_weapons.dm

```diff
@@ -594,6 +617,7 @@ TYPEINFO(/obj/item/sword/pink/angel)
 	throwforce = 20
 	stamina_cost = 5
 	c_flags = EQUIPPED_WHILE_HELD
+	can_stick_into_mob = FALSE
```

Comment:
Should probably apply to at least the syndie combat knife too since that otherwise works exactly the same as the dagger.

Reply:
If you think so, I was mostly just aiming at the syndicate dagger because of how it small it is, syndicate dagger is more clickable
![image](https://github.com/goonstation/goonstation/assets/53062374/4f267d92-71f7-46e1-89bf-6278351b8399)